internal/msgstream/mq_msgstream.go

```diff
@@ -66,8 +69,10 @@ func NewMqMsgStream(ctx context.Context,
 	streamCtx, streamCancel := context.WithCancel(ctx)
 	producers := make(map[string]mqclient.Producer)
 	consumers := make(map[string]mqclient.Consumer)
+	readers := make(map[string]mqclient.Reader)
 	producerChannels := make([]string, 0)
 	consumerChannels := make([]string, 0)
+	readerChannels := make([]string, 0)
```

Comment:
We need to change this to `var someChannels []string` later.

Reply:
lgtm